error: 'install-curseforge' command failed

### Describe the problem

when I run the docker to create the server, the container crashes and the server fails to start. 

RCON_PASSWORD and CF_API_KEY deleted from the container definition for security.

### Container definition


docker run -d -it --name mc-java \
   -v /home/gameservers/minecraft/java:/data \
   -v /home/gameservers/minecraft/java/downloads:/downloads \
   -e EULA=TRUE \
   -e MEMORY=32G \
   -e RCON_PASSWORD="" \
   -e SERVER_NAME="Animusreptor Crafting" \
   -e VERSION=1.18.2 \
   -e TYPE=AUTO_CURSEFORGE \
   -e CF_API_KEY='' \
   -e CF_PAGE_URL=https://www.curseforge.com/minecraft/modpacks/vault-hunters-1-18-2 \
   itzg/minecraft-server

### Container logs

[init] Running as uid=1000 gid=1000 with /data as 'drwxr-xr-x 7 1000 1000 4096 Oct 15 21:14 /data'

[mc-image-helper] 22:16:15.998 ERROR : 'install-curseforge' command failed. Version is 1.40.1

me.itzg.helpers.errors.GenericException: Failed to setup API caching: Unexpected end-of-input: expected close marker for Object (start marker at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 33])

 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 152] (through reference chain: me.itzg.helpers.cache.CacheIndex["operations"])

	at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:254)

	at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:191)

	at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:222)

	at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:30)

	at picocli.CommandLine.executeUserObject(CommandLine.java:2045)

	at picocli.CommandLine.access$1500(CommandLine.java:148)

	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2465)

	at picocli.CommandLine$RunLast.handle(CommandLine.java:2457)

	at picocli.CommandLine$RunLast.handle(CommandLine.java:2419)

	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)

	at picocli.CommandLine$RunLast.execute(CommandLine.java:2421)

	at picocli.CommandLine.execute(CommandLine.java:2174)

	at me.itzg.helpers.McImageHelper.main(McImageHelper.java:159)

Caused by: com.fasterxml.jackson.databind.JsonMappingException: Unexpected end-of-input: expected close marker for Object (start marker at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 33])

 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 152] (through reference chain: me.itzg.helpers.cache.CacheIndex["operations"])

	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:402)

	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:361)

	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.wrapAndThrow(BeanDeserializerBase.java:1964)

	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:312)

	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)

	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)

	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4917)

	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3725)

	at me.itzg.helpers.cache.ApiCachingImpl.loadCacheIndex(ApiCachingImpl.java:81)

	at me.itzg.helpers.cache.ApiCachingImpl.<init>(ApiCachingImpl.java:48)

	at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:222)

	... 12 common frames omitted

Caused by: com.fasterxml.jackson.core.io.JsonEOFException: Unexpected end-of-input: expected close marker for Object (start marker at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 33])

 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 152]

	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportInvalidEOF(ParserMinimalBase.java:641)

	at com.fasterxml.jackson.core.base.ParserBase._handleEOF(ParserBase.java:530)

	at com.fasterxml.jackson.core.base.ParserBase._eofAsNextChar(ParserBase.java:547)

	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._skipWSOrEnd(UTF8StreamJsonParser.java:3063)

	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser.nextFieldName(UTF8StreamJsonParser.java:999)

	at com.fasterxml.jackson.databind.deser.std.MapDeserializer._readAndBindStringKeyMap(MapDeserializer.java:608)

	at com.fasterxml.jackson.databind.deser.std.MapDeserializer.deserialize(MapDeserializer.java:449)

	at com.fasterxml.jackson.databind.deser.std.MapDeserializer.deserialize(MapDeserializer.java:32)

	at com.fasterxml.jackson.databind.deser.std.MapDeserializer._readAndBindStringKeyMap(MapDeserializer.java:623)

	at com.fasterxml.jackson.databind.deser.std.MapDeserializer.deserialize(MapDeserializer.java:449)

	at com.fasterxml.jackson.databind.deser.std.MapDeserializer.deserialize(MapDeserializer.java:32)

	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)

	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:310)

	... 19 common frames omitted

[init] ERROR failed to auto-install CurseForge modpack

--- a/Dockerfile
+++ b/Dockerfile
@@ -50,7 +50,7 @@ RUN easy-add --var os=${TARGETOS} --var arch=${TARGETARCH}${TARGETVARIANT} \
   --var version=${MC_SERVER_RUNNER_VERSION} --var app=mc-server-runner --file {{.app}} \
   --from ${GITHUB_BASEURL}/itzg/{{.app}}/releases/download/{{.version}}/{{.app}}_{{.version}}_{{.os}}_{{.arch}}.tar.gz
 
-ARG MC_HELPER_VERSION=1.40.1
+ARG MC_HELPER_VERSION=1.40.2
 ARG MC_HELPER_BASE_URL=${GITHUB_BASEURL}/itzg/mc-image-helper/releases/download/${MC_HELPER_VERSION}
 # used for cache busting local copy of mc-image-helper
 ARG MC_HELPER_REV=1

